prospector/tools/pylint/__init__.py

```diff
@@ -4,7 +4,10 @@
 import sys
 import os
 from pylint.config import find_pylintrc
-from pylint.utils import UnknownMessage
+try:
+    from pylint.utils import UnknownMessage
+except ImportError:
+    from pylint.utils import UnknownMessageError as UnknownMessage
```

Comment:
According to https://docs.pylint.org/en/latest/whatsnew/changelog.html#what-s-new-in-pylint-1-7:

> InvalidMessageError, UnknownMessage, and EmptyReport exceptions are moved to the new pylint.exceptions submodule.

So perhaps `from pylint.exceptions import UnknownMessageError` might be a better choice here.